lib/database/migrations/20240119143500-create-anchored_passes.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.createTable('anchored_passes', {
```

Comment:
As we discussed, I would rather name this table `simulation_pass_data_pass_anchors`, because `anchored_passes` are subsets of `simulation_passes`

Reply:
applied